openmetadata-spec/src/main/resources/json/schema/entity/services/connections/metadata/atlasConnection.json

```diff
@@ -39,22 +39,10 @@
       "description": "entity types of the data source.",
```

Comment:
Also for `entityTypes` I believe we are expecting a file path, when user will be doing this via UI putting file in a docker container could be a challenge, lets explore if we can eliminate this!